_studio/shared/umc/codec/h264_dec/include/umc_h264_task_supplier.h

```diff
@@ -734,7 +734,10 @@ inline int32_t CalculateDPBSize(uint8_t & level_idc, int32_t width, int32_t heig
         uint32_t dpbLevel = MaxDPBMbs*256 / (width * height);
         dpbSize = MFX_MIN(16, dpbLevel);
 
-        if (num_ref_frames <= dpbSize || level_idc == H264VideoDecoderParams::H264_LEVEL_MAX)
+        if (level_idc == H264VideoDecoderParams::H264_LEVEL_MAX)
+          dpbSize = MFX_MAX(dpbSize, num_ref_frames);
```

Comment:
I would ask to use ```std::max``` instead of macro. I know that this approach mimic existing style, but I want to get #925 merged someday, so it's better to not introduce potential conflicts

Reply:
Thanks for your comment, I updated the patch.